tardis/visualization/widgets/simulation.ipynb

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
<p>Generally all imports go into the topmost cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/3018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>